css/components/modal.css

```diff
@@ -0,0 +1,34 @@
+.modal {
+  width: 40rem;
+  padding: 2rem;
+  border-radius: 4px;
+  color: var(--white-color);
+  background: var(--dark-color);
+  position: relative;
+}
+.modal-title {
+  font-size: 2rem;
+  margin: 1rem 0;
+}
+.modal-text {
+  font-size: 1.3rem;
+  margin: 1rem 0;
+}
+.cross {
+  position: absolute;
+  top: 0.5rem;
+  right: .5rem;
+  padding: 0.2em 0.6em;
+  border-radius: 50%;
+  cursor: pointer;
+  border: 1px solid var(--white-color);
+}
+.cross:hover {
+  background-color: var(--white-color);
+  color: var(--dark-color);
+}
+.cancel-btn {
```

Comment:
Have you implemented a hover state for this button somewhere? If not, you can add it here.

Reply:
thanks for your suggestion